src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -314,6 +313,85 @@ public void TrackDependency(DependencyTelemetry telemetry)
             this.Track(telemetry);
         }
 
+        //// Preserve original version of 'Track' with TelemetryInitializer being called after iKey check for Devices SDK till we completely move to a different SDK.
+#if NET40 || NET45
```

Comment:
In 2.0-beta we will have only 1 method as it is now in master just with one line moved upper. Here because we cannot touch devices part it will be even harder to understand if we try to use "ifs". In old version we had: 1. check iKey and return 2. Call initializers 3. Call Send 4. Write to debug. In the new version we want 1. Call Initializers 2. Write to debug 3. Check iKey and return 4. Call send.